nemo/collections/multimodal/speech_llm/models/modular_t5_models.py

```diff
@@ -347,7 +348,7 @@ def prepare_llm_input(self, audio_batch):
         input_ids, input_length, labels, loss_mask = (
             audio_batch['contexts'],
             audio_batch['context_lengths'],
-            audio_batch['labels'],
+            audio_batch['answers'],
```

Comment:
any reason to update this ?

Reply:
Technically no, because it's unused anyway.